packages/react-components/src/components/tooltip/tooltip.css

```diff
@@ -0,0 +1,51 @@
+.tooltip {
+  position: relative;
+  display: inline-block;
+}
+
+.tooltip .tooltiptext {
+  position: absolute;
+  left: 50%;
+  top: 115%;
+  border-style: solid;
+  z-index: 9;
+  transform: translateX(-50%);
+  width: 120px;
+
+  /* margin-left: -60px;  */
+  visibility: hidden;
+  text-align: center;
+  padding: 5px 0;
+  border-radius: 6px;
+}
+
+/* Can't style pseudo selectors with inline styling */
+.tooltip .tooltiptext::before,
+.tooltip .tooltiptext::after {
```

Comment:
optional: for before and after worth using the event handlers and design tokens ?